server/command.go

```diff
@@ -145,6 +147,20 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 	ctx, cancel := context.WithTimeout(context.Background(), commandTimeout)
 	defer cancel()
 
+	defer func() {
+		if r := recover(); r != nil {
+			p.client.Log.Warn("Recovered from a panic",
+				"Command", args.Command,
+				"UserId", args.UserId,
+				"error", r,
+				"stack", string(debug.Stack()))
+			err = model.NewAppError("ExecuteCommand", "plugin_gitlab.command.execute_command.panic", nil, "", http.StatusInternalServerError)
+			res = &model.CommandResponse{}
+			res.Text = "An unexpected error occurred. Please try again later."
```

Comment:
If the server is running in Dev mode, it's fine to also post the error here. That makes it easier to debug issues.

Reply:
This can be determined by looking at `p.client.Config.Get().ServiceSettings.EnableDeveloper`